src/index.js

```diff
@@ -0,0 +1,129 @@
+import './style.css';
+
+// Ids
+const addList = document.getElementById('addList');
+const tasksContainer = document.getElementById('tasksContainer');
+const addInput = document.getElementById('addInput');
+const toDoSubmit = document.getElementById('toDoSubmit');
+const removeAll = document.getElementById('removeAll');
+
+let tasks = [];
+
+const setTask = () => {
+  const task = {
+    id: tasks.length + 1,
+    description: addInput.value,
+    completed: false,
+  };
+  tasks = [...tasks, task];
+};
+
+const printTasks = () => {
+  while (tasksContainer.firstChild) {
+    tasksContainer.removeChild(tasksContainer.firstChild);
+  }
+
+  if (tasks.length > 0) {
+    tasks.forEach((task) => {
+      const li = document.createElement('li');
+      li.classList.add('toDo__list__item');
+      li.innerHTML = `
+        ${task.completed ? (
+    `
+          <div class="toDo__list__itemContainer marked">
+          <i data-id="${task.id}" class="fa-solid fa-square-check completed"></i>
+          <label class="toDo__description checked">${task.description}</label>
+          </div>
+          `
+  ) : (
+    `
```

Comment:
- [ ] When I cloned your repo in my local machine and ran the project on local server. so when I try to add a to-do, it gives me an error. kindly consider fixing this error so that the code reviewer can check the functionalities properly.

https://github.com/Feliperto/ToDo_List/assets/96683843/6c106469-09f8-420a-bdae-64a843f3575f